src/pages/en/guides/cms/storyblok.mdx

```diff
@@ -420,12 +443,12 @@ This file will generate a page for each story, with the slug and content fetched
 
 If you’ve [opted into SSR mode](/en/guides/server-side-rendering/#enabling-ssr-in-your-project), you will use dynamic routes to fetch the page data from Storyblok.
```

Comment:
Just noting here that the previous instruction to create the directory and the file is techically under "if you're using SSG" so you might want a similar instruction (vs just a code sample that exists in comparison to the previous section) in case someone just skimmed and came directly here.